manifests/p/PhilipGerke/Odassey/2.0.0/PhilipGerke.Odassey.locale.en-US.yaml

```
@@ -1,24 +1,27 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
 PackageIdentifier: PhilipGerke.Odassey
-PackageVersion: 1.0.3
+PackageVersion: 2.0.0
 PackageLocale: en-US
-Publisher: Philip Gerke
+Publisher: Ingenieurbüro Philip Gerke
 PublisherUrl: https://www.philipgerke.com
-#PublisherSupportUrl: 
+PublisherSupportUrl: https://pgerke.atlassian.net/servicedesk/customer/portal/1
 #PrivacyUrl: 
 Author: Philip Gerke
 PackageName: Odassey
 PackageUrl: http://www.odassey.de
 License: Proprietary
-#LicenseUrl: 
-Copyright: 2020-2021 © Philip Gerke
+LicenseUrl: https://github.com/pgerke/get-odassey/blob/main/LICENSE.md
+Copyright: Copyright (c) 2020-2021 Philip Gerke
 #CopyrightUrl: 
 ShortDescription: Conveniently explore OData v4 endpoints
 Description: Make sure your journey on the OData ocean doesn't become an odyssey with Odassey
 Moniker: odassey
 Tags:
-  - odassey
-  - odata
-  - odata v4
+- odassey
+- odata
+- odata v4
```

The best practice is to present these tags in all lower case with hyphens rather than spaces. Referring to the 1.0.0 schema: [defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

```suggestion
- odata-v4
```